controllers/metal3.io/baremetalhost_controller_test.go

```diff
@@ -539,34 +539,34 @@ func TestDoNotAddSecretFinalizersDuringDelete(t *testing.T) {
 	// Let the host reach the available state before deleting it
 	waitForProvisioningState(t, r, host, metal3api.StateAvailable)
 	err := doDeleteHost(host, r)
-	assert.NoError(t, err)
+	require.NoError(t, err)
```

Comment:
do we have to change all `asserts` to `require` ? is it something that Linter is forcing?

Reply:
It is not changing all of them, just the ones `NoError`. `NoError` is expected to be `fatal` failure that will ruin the rest of test run anyways, if it the assertion fails, so require just calls it a day earlier?